includes/options/class-amp-options-manager.php

```diff
@@ -327,6 +328,45 @@ public static function update_analytics_options( $data ) {
 		return self::update_option( 'analytics', wp_unslash( $data ) );
 	}
 
+	/**
+	 * Renders the welcome notice on the 'AMP Settings' page.
+	 *
+	 * Uses the user meta values for the dismissed WP pointers.
+	 * So once the user dismisses this notice, it will never appear again.
+	 *
+	 * @todo: Add the full copy for this notice when it is decided.
+	 */
+	public static function render_welcome_notice() {
+		if ( 'toplevel_page_' . self::OPTION_NAME !== get_current_screen()->id ) {
+			return;
+		}
+
+		$notice_id = 'welcome-notice-1';
+		$dismissed = get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true );
+		if ( in_array( $notice_id, explode( ',', strval( $dismissed ) ), true ) ) {
```

Comment:
**Question About Admin Pointer User Meta**

Hi @westonruter,
What do you think about how this uses the user meta for admin pointers to keep track of whether this notice was dismissed? 

This is very similar to how [this admin pointer PR](https://github.com/Automattic/amp-wp/pull/1271/files#diff-7ec3f55e1310503aa82ce61984d3f482R78) does this.

Of course, this isn't actually an admin pointer, so it's not ideal. 

But this is a little simpler, as it can reuse the `dismiss-wp-pointer' action handler when the user dismisses it.

Thanks, Weston!

Reply:
My main concern is what happens over time. If each new version of the plugin will include new information that is shown in this message, then the `dismissed_wp_pointers` usermeta will grow endlessly.

You could instead consider what was added for the Try Gutenberg screen:

https://github.com/WordPress/wordpress-develop/blob/5d477aa70613850a2e78cdaecb7e2de365de7134/src/wp-admin/js/dashboard.js#L34-L75

https://github.com/WordPress/wordpress-develop/blob/5d477aa70613850a2e78cdaecb7e2de365de7134/src/wp-admin/includes/ajax-actions.php#L1487-L1498

But maybe this is all overkill. For a one-off, I don't see a reason why to not use the admin-pointers. They're being used for non-pointer things already, including the theme and plugin editor warnings:

https://github.com/WordPress/wordpress-develop/blob/5d477aa70613850a2e78cdaecb7e2de365de7134/src/wp-admin/plugin-editor.php#L282-L294

https://github.com/WordPress/wordpress-develop/blob/5d477aa70613850a2e78cdaecb7e2de365de7134/src/wp-admin/theme-editor.php#L301-L313

https://github.com/WordPress/wordpress-develop/blob/5d477aa70613850a2e78cdaecb7e2de365de7134/src/wp-admin/js/theme-plugin-editor.js#L134-L150

So I'd say go ahead with pointers.